autosynth/multi.py

```diff
@@ -272,7 +272,7 @@ def load_config(
     """
     if os.path.exists(config):
         with open(config) as fh:
-            return yaml.load(fh)["libraries"]
+            return yaml.load(fh, Loader=yaml.FullLoader)["libraries"]
```

Comment:
Why was this change necessary?

Reply:
This was in response to your last comment to fix lint warnings. It was being flagged in the logs in the kokoro job.

Reply:
😳